custom_components/zaptec/api.py

```diff
@@ -447,11 +454,21 @@ async def build(self):
         self.installs = cls_installs
 
         so_chargers = await self.chargers()
+        device_types = []
         for charger in so_chargers:
             if charger.id not in self.map:
                 self.map[charger.id] = charger
+
+            # Append the device type in order to read the extended
+            # IDs from the remap dict.
+            device_types.append(charger.device_type)
```

Comment:
It collects all present chargers in order to determine which device specific schemas to include.